pkg/util/system/common_linux.go

```diff
@@ -36,6 +36,24 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// Port2Pid Query pid by port number with the help of lsof command
+func Port2Pid(port int) (int, error) {
+	cmd := exec.Command("/bin/sh", "-c", "lsof -i:"+strconv.Itoa(port)+" | awk {'print$2'} | awk 'NR==2{print}'")
```

Comment:
we don't support `lsof` in our current docker image right now. any other methods?
![image](https://user-images.githubusercontent.com/24452340/186403914-47491fbd-3e3e-4dd6-bbd0-862c4b9aaa2c.png)